test/ansible-inventory/molecule/test-local/playbook.yml

```diff
@@ -24,7 +24,7 @@
     deploy_dir: "{{ lookup('env', 'MOLECULE_PROJECT_DIRECTORY') }}/deploy"
     pull_policy: Never
     REPLACE_IMAGE: inventory.apps.fabianism.us/inventory:testing
-    custom_resource: "{{ lookup('file', '/'.join([deploy_dir, 'crds/inventory_v1alpha1_inventory_cr.yaml'])) | from_yaml }}"
+    custom_resource: "{{ lookup('file', '/'.join([deploy_dir, 'crds/inventory.apps.fabianism.us_v1alpha1_inventory_cr.yaml'])) | from_yaml }}"
```

Comment:
Nit and unrelated: should we change the domain to `example.com`?

Reply:
@fabianvf ok with this? we can change it in a follow-up PR